packages/common/src/tokens/utils.test.ts

```diff
@@ -0,0 +1,30 @@
+import { Token } from './token';
+import { expect } from 'chai';
+import { mainnetTokens } from './basicTokens';
+import { unifyTokens } from './utils';
+
+describe('unifyTokens', () => {
+  it('should replace property if input token is basic token', () => {
+    const chainId = 1;
+    const tokenSet: [number, string, number, string, string, string][] = [
+      [1, '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48', 6, 'USDC', 'UUUSDC', 'some url'],
```

Comment:
```suggestion
    const tokenSet = [
      new Token(1, '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48', 6, 'USDC', 'UUUSDC', 'some url'),
```

Reply:
fixed, also put the tokens to outer scope